rllib/env/single_agent_env_runner.py

```diff
@@ -494,12 +497,14 @@ def _sample_episodes(
 
                 # RLModule forward pass: Explore or not.
                 if explore:
-                    env_steps_lifetime = (
+                    # Global env steps sampled are (roughly) this EnvRunner's lifetime
```

Comment:
EnvRunners do NOT carry the global timesteps anymore. Instead, they only keep track of their very own lifetime counts and estimate the global lifetime steps through simply multiplying with the number of EnvRunners in the algo. This way, we can simplify and separate concerns even more and a) skip the extra broadcast of global timesteps from algo back to all workers and b) more naturally "reduce" the lifetime steps on the algo side by simply adding over the individual EnvRunners' results.

Reply:
@sven1977 This means that epsilon greedy will now depend on a single runner's env steps sampled over its lifetime and decay slower. Did we already test in DQN what influence this has?

Reply:
I think it's fine. The EnvRunners are now doing this here:
```
global_env_steps_lifetime = (
                        self.metrics.peek(NUM_ENV_STEPS_SAMPLED_LIFETIME, default=0)
                        + ts
                    ) * (self.config.num_env_runners or 1)  <- !!HERE!! (multiply by number of env runners to get a rough estimate of the overall timesteps of all env-runners
```